src/dynamic_programming/intro-to-dp.md

```diff
@@ -130,19 +130,21 @@ That's it. That's the basics of dynamic programming: Don't repeat work you've do
 One of the tricks to getting better at dynamic programming is to study some of the classic examples.
 
 ## Classic Dynamic Programming Problems
-- 0-1 Knapsack
-- Subset Sum
-- Longest Increasing Subsequence
-- Counting all possible paths from top left to bottom right corner of a matrix
-- Longest Common Subsequence 
-- Longest Path in a Directed Acyclic Graph (DAG)
-- Coin Change
-- Longest Palindromic Subsequence
-- Rod Cutting
-- Edit Distance
-- Bitmask Dynamic Programming
-- Digit Dynamic Programming
-- Dynamic Programming on Trees
+| Name                                           | Description                                                                                                                                                                                      | Example Poblems                                                                                                                                 |
+| ---------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | ----------------------------------------------------------------------------------------------------------------------------------------------- |
+| 0-1 Knapsack                                   | 0-1 Knapsack                                                                                                                                                                                     | given a set of items, each with a weight and a value, and a knapsack with a maximum capacity.                                                   | [D - Knapsack 1 (](https://atcoder.jp/contests/dp/tasks/dp_d)[atcoder.jp](http://atcoder.jp)[)](https://atcoder.jp/contests/dp/tasks/dp_d) |
+| Subset Sum                                     | you are given a set of integers and a target sum. The task is to determine whether there exists a subset of the given set whose elements sum up to the target sum.                               |                                                                                                                                                 |
```

Comment:
Probably should have uninformed descriptions like: Given a, b, and c, find z.

Reply:
Fair enough.

Reply:
Let's make it simple. Two columns, one for name, the other for description